int32[] obj_lane
float32[] obj_track_distance
float32[] obj_lateral_offset

float32[] pedestrian_lateral_offset

int32[] traffic_marks_id
float32[] traffic_marks_track_distance